src/app/reporting/Engine.cpp

```diff
@@ -99,17 +97,39 @@ CHIP_ERROR Engine::BuildSingleReportDataAttributeDataList(ReportData::Builder &
     // TODO: Need to handle multiple chunk of message
     while (clusterInfo != nullptr)
```

Comment:
The same for this one, the `cluster = clusterInfo->mpNext` is 30 lines below, and I have to scoll my screen to see it.

Reply:
Done, thanks